test/e2e/index.test.ts

```diff
@@ -276,6 +276,38 @@ describe('e2e', () => {
 
       expect(pageErrors.length).to.equal(0);
     });
+
+    it('should not glitch when resizing', async () => {
+      await renderFixture('TextareaAutosize/BasicTextareaAutosize');
+
+      const textarea = await screen.getByTestId('textarea')!;
+
+      // Get the element's dimensions
+      const { x, y, width, height } = (await textarea.boundingBox())!;
+
+      // Calculate coordinates of bottom-right corner
+      const bottomRightX = x + width;
+      const bottomRightY = y + height;
+
+      // Get the initial height of textarea as a number
+      const initialHeight = await textarea.evaluate((event) => parseFloat(event.style.height));
+
+      // Move the mouse to the bottom-right corner, adjusting slightly to grab the resize handle
+      await page.mouse.move(bottomRightX - 5, bottomRightY - 5);
+
+      // Simulate a double click without releasing the mouse button (mouseup) to grab the resize handle
```

Comment:
Is there a particular reason a double click is needed?

Reply:
If you test it manually in the documentation preview you'll see that a double-click is necessary to grab the handle for resizing. However, it's essential not to release the mouse button after the second click. Simply clicking once with the mouse doesn't work. I'm using the touchpad on my laptop.